inception-project/inception
# #1771 Hide Agreement menu item if not enough users

Same as https://github.com/webanno/webanno/issues/1735

**What's in the PR**
* check in agreement menuitem if there are more than one annotators in the project


**How to test manually**
* Be the only user in a project
* Open the dashboard, you should not see the agreement menu item
* Add another user to the project, you should now see the agreement item
